src/client/Microsoft.Identity.Client/Platforms/Features/WamBroker/AadPlugin.cs

```diff
@@ -26,12 +26,14 @@ internal class AadPlugin : IWamPlugin
         private readonly IWamProxy _wamProxy;
         private readonly IWebAccountProviderFactory _webAccountProviderFactory;
         private readonly ICoreLogger _logger;
+        private readonly Dictionary<string, string> _telemetryHeaders;
 
         public AadPlugin(IWamProxy wamProxy, IWebAccountProviderFactory webAccountProviderFactory, ICoreLogger logger)
         {
             _wamProxy = wamProxy;
             _webAccountProviderFactory = webAccountProviderFactory;
             _logger = logger;
+            _telemetryHeaders = new Dictionary<string, string>(MsalIdHelper.GetMsalIdParameters(logger));
```

Comment:
GetMsalIdParameters already returns an IDictionary, why do you need another one? If you need `Dictionary`, change `GetMsalIdParameters`

Reply:
Done.